pyjelly/integrations/generic/serialize.py

```diff
@@ -234,7 +235,11 @@ def guess_options(sink: GenericStatementSink) -> SerializerOptions:
         if sink.is_triples_sink
         else jelly.LOGICAL_STREAM_TYPE_FLAT_QUADS
     )
-    return SerializerOptions(logical_type=logical_type)
+    params = StreamParameters(
```

Comment:
Please leave a comment here explaining why we set it to true, true. This will help people understand the code.

Same goes for the rdflib version.